docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -0,0 +1,190 @@
+
+:doc:`ClassPerformance </checks_gallery/vision/model_evaluation/plot_class_performance>`:
+
+.. code-block:: python
+
+    check = ClassPerformance()
+    result = check.run(train_ds, test_ds, mnist_model)
+
+
+Alternative metrics
+===================
+Sometimes the defaults don't fit the specifics of the use-case.
+If this is the case, you can pass a list of supported metric strings or a dict in the format {metric name: metric} as a
+parameter to the check.
+
+The metrics in the dict can be existing:
+
+*   `Ignite metrics <https://pytorch.org/ignite/metrics.html#complete-list-of-metrics>`__ for vision.
+    An Ignite Metric is a class with the methods: reset, compute, and update, that iterates over batches of data and
+    aggregates the result.
+*   `Sklearn scorers <https://scikit-learn.org/stable/modules/model_evaluation.html>`__ for both vision and tabular.
+    An Sklearn Scorer is a function that accepts the parameters: (model, x, y_true), and returns a score with the
+    convention that higher is better.
+*   strings from Deepchecks' `supported strings <#list-of-supported-strings>`__ for both vision and tabular.
+
+or `your own implementation <#custom-metrics>`__.
+
+.. code-block:: python
+
+   train_ds, test_ds = adult.load_data(data_format='Dataset', as_train_test=True)
+   model = adult.load_fitted_model()
+
+   scorer = ['precision_per_class', 'recall_per_class', 'fnr_macro']
+   check = TrainTestPerformance(scorers=scorer)
+   result = check.run(train_ds, test_ds, model)
+   result
+
+
+List of Supported Strings
+=========================
+.. list-table:: Supported Strings
+   :widths: 25 75 75
+   :header-rows: 1
+
+   * - String
+     - Metric
+     - Comments
+   * - 'neg_rmse'
+     - negative root mean squared error
+     - Sklearn
+   * - 'neg_mae'
+     - negative mean absolute error
+     - Sklearn
+   * - 'precision_per_class'
+     - precision per class - no averaging (results are returned per class)
+     -
+   * - 'average_precision_per_class'
+     - average precision for object detection
+     - only for object detection
+   * - 'recall_per_class' , 'sensitivity_per_class'
+     - recall per class - no averaging
+     -
+   * - 'average_recall_per_class'
+     - average recall for object detection
+     - only for object detection
+   * - 'f1_per_class'
+     - F-1  per class - no averaging
+     -
+   * - 'fpr_per_class'
+     - False Positive Rate per class - no averaging
+     -
+   * - 'fpr'
+     - False Positive Rate - binary
+     -
+   * - 'fpr_macro'
+     - False Positive Rate - macro averaging
+     -
+   * - 'fpr_micro'
+     - False Positive Rate - micro averaging
+     -
+   * - 'fpr_weighted'
+     - False Positive Rate - weighted macro averaging
+     -
+   * - 'fnr'
+     - False Negative Rate - binary
+     - suffixes apply as with 'fpr'
+   * - 'tnr', 'specificity'
+     - True Negative Rate - binary
+     - suffixes apply as with 'fpr'
+
+
+In addition to the strings listed above, all Sklearn
```

Comment:
Maybe put this comment at the head of this section

Reply:
+1